docs/data/joy/getting-started/tutorial/LoginFinal.js

```diff
@@ -6,7 +6,7 @@ import TextField from '@mui/joy/TextField';
 import Button from '@mui/joy/Button';
 import Link from '@mui/joy/Link';
 
-const ModeToggle = () => {
+function ModeToggle() {
```

Comment:
I think the convention we use in the docs to create components. It's a bit related to this PR https://github.com/mui/material-ui/pull/34644/files#diff-559bb65f49006b76c9495639f8c69571711721a76d0a19ae279ca4f93772e6ffR9